src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -21978,7 +21978,7 @@ class C
                 Diagnostic(ErrorCode.WRN_DotOnDefault, "default(T1)[1]").WithArguments("T1").WithLocation(35, 9),
                 // (37,9): error CS0131: The left-hand side of an assignment must be a variable, property or indexer
                 Diagnostic(ErrorCode.ERR_AssgLvalueExpected, "default(T3)[1]").WithLocation(37, 9), // Incorrect? See CS0131ERR_AssgLvalueExpected03 unit test.
-                // (38,9): warning CS1720: Expression will always cause a System.NullReferenceException because the default value of 'T4' is null
+                                                                                                    // (38,9): warning CS1720: Expression will always cause a System.NullReferenceException because the default value of 'T4' is null
```

Comment:
i thought we fixed this.

Reply:
💭 I don't remember a fix for this

Reply:
yeah, i'm not sure if it was an actual fix, or just a bug+design-decision.  Specifically, that we would not try to realign the comment on the next line if it aligned with the construct following it.  But i don't remember what came of htat.

Reply:
➡️ Yes, that was correct. The open issue (and specifically the comment describing the desired result) is here: https://github.com/dotnet/roslyn/issues/29482#issuecomment-481007848